### DISCUSSION TITLE
https://steinkamp.us/posts/2022-03-16-knobbler

### DISCUSSION THREAD

**Discussion by giscus** [zsteinkamp/steinkamp.us#17] [Announcements]
# https://steinkamp.us/posts/2022-03-16-knobbler

https://steinkamp.us/posts/2022-03-16-knobbler

**Comment by hansolero** [comment]
You are awsome 🙌😊
i hope you can see my comment! i just registered to github to thank you for your work. keep it up i like all of your work 🙌😊🙌

**Reply by zsteinkamp** [reply]
Thank you @hansolero :) I'm happy to hear your feedback on it. I just released Knobbler4 which brings better performance and reliability. https://github.com/zsteinkamp/m4l-Knobbler4

**Comment by spoeokr** [comment]
I'm fascinated by this and going to try it first thing in the morning. Thank you for making these amazing devices.

**Reply by zsteinkamp** [reply]
Happy to hear about your experiences with it and opportunities to make it better :) zack@steinkamp.us
